### ISSUE TITLE
mpv does not go into full screen mode

### ISSUE THREAD

**Issue by ATVVA** [JLErvin/berry#84]
When press the "f" key, the video does not go into full screen mode. Also, if run mpv with the _--fs_ option, the video is not full screen.
```
$ berry -v
berry 0.1.5
Copyright (c) 2018 Joshua L Ervin
Released under the MIT License

$ mpv -V                               
mpv 0.32.0 Copyright © 2000-2020 mpv/MPlayer/mplayer2 projects
 built on UNKNOWN
ffmpeg library versions:
   libavutil       56.31.100
   libavcodec      58.54.100
   libavformat     58.29.100
   libswscale      5.5.100
   libavfilter     7.57.100
   libswresample   3.5.100
ffmpeg version: n4.2.2
```

**Comment by JLErvin** [comment]
I was able to recreate this on my end. Will get back to you shortly.

**Comment by JLErvin** [comment]
Added to the development branch as of 7c2507c9a5fc5e82dde2edfc7cc670118d14fed9. You can try pulling/cloning from there and see if it resolves your issue. I'll close this issue once this is merged into master.

**Comment by JLErvin** [comment]
Added as of 0.1.6. Closing.
